### DISCUSSION TITLE
Prisma deleted all data from the database after  migrate dev --name init

### DISCUSSION THREAD

**Discussion by cakctus** [prisma/prisma#16950] [Q&A]
I connected a third-party database through prism db push, after for example I want to make some changes to an existing database, add some row in the Post Model for exemple, and apply migrations using prisma migrate dev. After that, the prism deleted all data. How can I change the data after prisma migrate dev ? My [prisma scheme](https://github.com/cakctus/motorsfy-backend/blob/master/prisma/schema.prisma)

**Comment by nurul3101** [comment]
Hey @cakctus 👋 

Prisma will not automatically delete the data, it will explicitly ask for your approval to delete the data in case a drift is detected.
Can you please elaborate on what you are trying to achieve?

For reference, here's the documentation of [Prisma Migrate](https://www.prisma.io/docs/guides/database/developing-with-prisma-migrate).

**Reply by cakctus** [reply]
I have a django application, I want to migrate to nodejs. I synchronized data from Postgresql with prisma using npx prisma db pull. But I want to change the models to prisma and store in the database, I use the npx prisma migrate dev command and I get the answer "We need to reset the PostgreSQL database "mydb" at "localhost:5432".
Do you want to continue? All data will be lost. » (y/N)" But I don't want to delete the data, but just want to make some changes, for example, add another model or add or remove one field from the model and store the data in the database

**Reply by nurul3101** [reply]
Did you have a look at [Baselining](https://www.prisma.io/docs/guides/database/developing-with-prisma-migrate/baselining) docs?  It looks like you need to Baseline your database.

**Reply by cakctus** [reply]
No, I skipped this section. Thanks

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
